feature-detects/mediastream.js

```diff
@@ -0,0 +1,22 @@
+/*!
+{
+  "name": "MediaRecorder API",
+  "property": "mediarecorder",
+  "caniuse": "mediarecorder",
+  "tags": ["mediarecorder"],
+  "authors": ["Jaikishan Brijwani"],
+  "notes": [{
+    "name": "MDN Docs",
+    "href": "https://developer.mozilla.org/en-US/docs/Web/API/MediaStream_Recording_API"
+  }],
+  "polyfills": ["MediaStream Recording API", "MediaStreamRecordingAPI"]
```

Comment:
Okay, everything in the metadata is fine except for the polyfills section (not a big problem). When you declare a polyfill you have to do an special process so the website can provide a link to download. So if you have any script with a website in mind I leave you the documentation for polyfills's link; https://github.com/Modernizr/Modernizr/blob/master/HOW_TO_WRITE_FEATURE_DETECTS.md#polyfills

However, I feel like you really don't have any scripts in mind, so if that's the case just delete the polyfill section from the metadata and the build should be okay.

Extra: If you want to verify that no errors are presented before you upload the code you can run `npm test` inside the Modernizr folder. Note that you have to have [npm&node installed](https://nodejs.org/) and that you must run `npm install` before.